lib/tasks/wraith.rake

```diff
@@ -0,0 +1,17 @@
+require 'pry'
+require 'rest-client'
+
+desc "check format in wraith."
+task :wraith_format, [:format] do |t, args|
+  format = args[:format]
+  wraith = YAML::load(File.open('test/wraith/config.yaml'))
+  search = RestClient.get "https://www.gov.uk/api/search.json?filter_content_store_document_type=#{format}&count=10"
```

Comment:
`filter_content_store_document_type` returns `document_type` not `schema_name`.
Can the task be called "check document type", and format replaced with `document_type` throughout?

eg. `specialist_document` is a format – it's a schema name, but it returns no results: https://www.gov.uk/api/search.json?filter_content_store_document_type=specialist_document&count=10

`cma_case` is a document_type, one of the allowed values for specialist_document schemas, which returns results:
https://www.gov.uk/api/search.json?filter_content_store_document_type=cma_case&count=10

Format is deprecated in schema.